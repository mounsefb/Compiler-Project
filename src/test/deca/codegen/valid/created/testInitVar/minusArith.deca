// Description :
//    Test de l'affichage d'une variable float résultant d'une opération arithmétique
//
// Résultats :
//    OK


{
    float y = -5.3 * 7;
    println(y);
}


