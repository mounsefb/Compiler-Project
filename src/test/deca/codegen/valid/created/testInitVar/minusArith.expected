-3.71000e+01
